OneCommunityGlobal/HighestGoodNetworkApp
# Ishan -- > Tapan - fix timelogs tab reverting to user instead of staying on person you are viewing

# Description
<img width="946" alt="Screenshot 2024-08-01 at 11 22 11 PM" src="https://github.com/user-attachments/assets/4950b75b-88db-4b60-93c7-592e5bebbb94">


https://www.loom.com/share/f650f6233040480b9b2931736fa1dde8?sid=5b7a0f00-50c8-4a48-9863-d7b77932eec8

## Related PRS (if any):
N/A
…

## Main changes explained:
-  Fix timelogs tab reverting to user instead of staying on person you are viewing
- Fixed Code in Timelog to get the url from the link
…

## How to test:
1. check into current branch
2. do `npm install` and `...` to run this PR locally
3. Clear site data/cache
4. log as admin/owner user
5. Go to Leaderboard ->  Click on Progress Bar of Any Volunteer
6. Go to Leaderboard -> Click on Any Volunteer -> Click the timelog button on their dashboard
7. Observe time log doesn't revert back to logged in user timelog even after waiting a few seconds
8. Try step 5, 6 and 7 back to back for multiple volunteers and see bug doesn't appear

## Screenshots or videos of changes:
Before - 
https://www.loom.com/share/f650f6233040480b9b2931736fa1dde8?sid=5b7a0f00-50c8-4a48-9863-d7b77932eec8

After - 

https://github.com/user-attachments/assets/b37cca46-0da2-42cc-a75c-62bf7e87695b
